docs/application/dotnet/guides/nui/nui-components/RadioButton.md

```diff
@@ -0,0 +1,78 @@
+# RadioButton
+RadioButton is a common component and describes what action will occur when you select it.  
+A RadioButton can only contain an icon, and can be created using style.
+
+![RadioButton1](./media/RadioButton1.png)
+
+## Create with Style
+
+To create a RadioButton applying style, follow these steps:
+
+1. Create RadioButton using the default constructor:
+
+    ```cs
+    RadioButton utilityRadioButton = new RadioButton();
+    ```
+
+2. Apply style to the RadioButton:
+
+    ```cs
+    ButtonStyle utilityStyle = new ButtonStyle
+    {
+        Icon = new ImageViewStyle
+        {
+            Size =  new Size(48, 48),
+            ResourceUrl = new Selector<string>
+            {
+                Normal = CommonResource.GetFHResourcePath() + "9. Controller/controller_btn_radio_off.png",
+                Selected = CommonResource.GetFHResourcePath() + "9. Controller/controller_btn_radio_on.png",
+                Disabled = CommonResource.GetFHResourcePath() + "9. Controller/controller_btn_radio_off.png",
+                DisabledSelected = CommonResource.GetFHResourcePath() + "9. Controller/controller_btn_radio_on.png",
```

Comment:
Just for checking. The image names and the paths are unusual. Would you please confirm them?

- `9. Controller/controller_btn_radio_off.png`
- `9. Controller/controller_btn_radio_on.png`

How about changing the code to show only the image names, so that it doesn't give any further information about your local work space?

Reply:
Fixed